Playground/src/components/commandBarComponent.tsx

```diff
@@ -74,6 +75,7 @@ export class CommandBarComponent extends React.Component<ICommandBarComponentPro
         var engineOptions = [
             {
                 label: "WebGL2",
+                tooltip: "Use WebGL 2 API",

```

Comment:
I would say renderer instead of API maybe?

Reply:
done :)